source/stylesheets/_layout.scss

```diff
@@ -0,0 +1,45 @@
+@import "bourbon";
+@import "settings/grid_layout";
+@import "neat";
+
+.page-header > .container {
+  @include outer-container(960px);
```

Comment:
I meant, if it's even necessary to specify a width. What high-end features are you talking about? The zooming boxes? ;)